installers/go-agent/release/go-agent.init

```diff
@@ -96,7 +95,7 @@ stop_go_agent() {
     check_proc
 
     if [ $? -eq 0 ]; then
-        killproc -p $PID_FILE /usr/share/${SERVICE_NAME}/agent.sh >/dev/null
+        killproc /usr/share/${SERVICE_NAME}/stop-agent.sh ${SERVICE_NAME} service_mode
```

Comment:
My vote would be to just explicitly invoke `stop-agent.sh` and remove `killproc()` altogether since it's now a useless use of `eval`.

Suggesting to change this line to:

```
>/dev/null /usr/share/${SERVICE_NAME}/stop-agent.sh ${SERVICE_NAME} service_mode
```